feat: Updating integrations section (#536)
apify/apify-docs: sources/platform/integrations/make.md

---
title: Make

> description: Learn how to integrate your Apify actors with Make.

Reviewer: ```suggestion
description: Learn how to integrate your Apify Actors with Make.
```

sidebar_position: 11.12
slug: /integrations/make
---

# Make integration

**Learn how to integrate your Apify actors with Make.**

---

> [Make](https://make.com/) *(formerly Integromat)* allows you to create scenarios where you can integrate various services (modules) to automate and centralize jobs. Apify has its own module which can be used to run Apify actors, be notified about run status, and get actor results directly in your Make scenario.

Reviewer: ```suggestion
[Make](https://make.com/) *(formerly Integromat)* allows you to create scenarios where you can integrate various services (modules) to automate and centralize jobs. Apify has its own module you can use to run Apify Actors, get notified about run statuses, and receive Actor results directly in your Make scenario.
```
